test-common/spawn-utils.c

```diff
@@ -96,6 +96,20 @@ cmd_result_ensure_all_ok_verbose (GPtrArray *cmds)
   return ok;
 }
 
+void
+cmd_results_allow_failure_verbose (GPtrArray *cmds)
+{
+  guint idx;
```

Comment:
Nitpick: Could move the definition of `idx` into the `for` loop header.

Reply:
Done